kinder/pkg/cluster/manager/actions/kubeadm-init.go

```diff
@@ -40,9 +40,9 @@ import (
 func KubeadmInit(c *status.Cluster, usePhases, kubeDNS, automaticCopyCerts bool, kustomizeDir, patchesDir string, wait time.Duration, vLevel int) (err error) {
 	cp1 := c.BootstrapControlPlane()
 
-	// fail fast if required to use kustomize and kubeadm less than v1.16
-	if kustomizeDir != "" && cp1.MustKubeadmVersion().LessThan(constants.V1_16) {
-		return errors.New("--kustomize-dir can't be used with kubeadm older than v1.16")
+	// fail fast if required to use kustomize and kubeadm less than v1.17
+	if kustomizeDir != "" && cp1.MustKubeadmVersion().LessThan(constants.V1_17) {
+		return errors.New("--kustomize-dir can't be used with kubeadm older than v1.17")
```

Comment:
once the 1_16 constant is removed we can remove the whole check with the assumption that 1.16 test jobs will not run.

Reply:
Done.